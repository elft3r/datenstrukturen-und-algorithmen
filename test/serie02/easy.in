100
82 90 5 7 74 -86 -71 -22 35 -77 -37 72 -97 -11 75 51 49 -38 54 9 -66 66 39 57 71 98 10 -80 84 -16 -34 1 -64 -93 -30 7 79 -58 41 46 -72 -10 21 28 -52 -5 71 100 36 29 34 -44 -42 34 -11 86 58 -94 -44 53 88 18 -48 44 48 44 -40 -80 -21 -28 13 -30 96 73 -60 -98 45 -26 -77 -76 80 -80 82
87 45 0 78 -97 15 12 -50 51 -82 84 -39 -72 70 6 -57 100 84 50 38 25 -68 -20 22 -85 40 16 16 -72 -59 34 79 77 -45 30 24 -24 35 78 -100 -76 -81 -83 87 5 -53 -63 -83 84 77 -41 58 -39 -82 -39 -85 -6 100 -42 -86 -3 -29 -20 98 8 86 12 -92 -3 -51 -69 -88 -86 100 24 -14 -80 -37 -80 86 -75 80 28 62 90 98 -40 18
77 84 57 -95 99 80 -28 37 13 87 57 93 100 -61 -26 39 -32 23 -66 55 -35 48 -23 7 -84 1 -58 -39 77 -89 -74 56 74 -76 47 -69 100 94 12 86 -2 94 0 -2 -90 47 26 18 75 35 -8 51 -14 -13 18 34 65 -2 45 81 -37 16 19 -8 62 2 -54 75 -41 99 93 -45 66 -37 -16 -25 -99 -30
38 -30 23 75 -88 95 -25 -69 74 72 18 -77 -93 6 -32 -61 -41 -69 89 -22 94 24 -54 -45 75 -6 27 83 -44 -38 -53 86 14 17 54 -80 -11 -67 21
15 22 13 -84 60 60 -94 43 95 34 78 90 44 -19 10 -29
42 85 23 17 60 17 -57 -84 0 -21 32 -36 -75 -6 -81 -13 88 -27 -18 82 -51 100 87 16 -80 -39 15 15 -6 -75 63 2 95 51 6 14 -74 -33 -60 76 -80 -1 -84
84 48 -88 45 -91 0 14 52 40 89 42 -76 -91 -54 96 -100 -46 -29 23 71 -63 -80 -74 -10 -97 89 -74 -39 60 -57 3 -62 28 -59 46 -1 -12 47 24 -7 -17 -15 -30 -53 -28 -32 -49 37 -4 18 23 8 -8 -63 11 21 82 -87 70 87 -50 -50 -65 62 -64 -13 64 31 6 -87 3 34 -87 -2 -35 96 49 -84 -94 -38 23 -80 -1 53 21
65 60 -91 41 53 -54 72 66 12 6 -94 0 96 -25 -5 -86 72 -11 -13 -19 -63 20 -30 -63 -28 80 -36 2 -51 -91 -57 -52 22 -83 22 -71 -81 58 -55 -30 -7 74 -79 22 69 -62 -87 -63 -94 11 -33 86 -92 -33 58 -87 -10 89 31 -42 -96 50 32 -43 38 -72
72 13 34 35 81 82 96 16 -29 0 -29 -60 68 -36 -68 97 -83 77 -26 -35 -21 24 89 60 -78 -61 17 39 82 -78 -51 -6 49 97 74 15 12 -10 57 -13 -80 -52 -22 -89 25 -52 28 20 14 59 23 10 39 36 -23 -33 9 1 15 83 -54 -74 18 12 -29 92 63 -100 66 6 17 -93 -65
23 -89 -20 -31 78 -39 -1 -82 26 28 81 -29 63 93 -19 -61 100 -93 -55 47 -12 35 -7 73
66 60 -45 43 -63 96 100 -31 -49 -65 -48 -66 -65 92 88 41 6 -61 -98 -31 -8 79 -74 -47 5 -36 -49 95 7 93 47 -40 26 -20 89 75 25 51 -63 -35 -95 -40 -61 -75 6 81 -54 64 35 -64 -36 40 -13 60 -78 -51 94 -15 -2 -95 -64 -43 47 -76 41 68 99
93 86 -79 62 -59 87 -40 71 -60 -95 18 -100 26 -40 94 35 -25 -38 24 76 56 -80 15 91 -87 -17 59 -18 7 76 59 43 62 -51 31 -39 -80 16 -87 76 39 -90 -55 12 -86 4 98 -50 -3 -41 83 70 83 83 -34 53 94 -86 -95 -88 1 -36 -41 -64 -14 -23 55 83 8 48 -73 81 25 11 85 9 86 16 -50 60 32 76 -55 25 22 -68 -49 12 -71 -15 -42 -17 -14 1
91 -91 -28 -69 -23 -63 -39 23 34 90 -56 45 4 100 -58 2 -49 26 -80 -5 -54 57 20 41 52 -53 14 58 49 80 16 45 72 -21 -23 75 24 83 -30 100 74 -23 58 57 -69 91 -58 -65 88 -22 99 -14 9 29 61 -99 -73 -75 99 -51 -16 42 63 -31 78 47 -25 28 -74 58 -1 18 44 38 26 -75 -38 37 35 -98 -38 -52 -43 79 39 1 -2 34 -69 28 -41 -47
44 -40 -23 -64 -95 17 86 51 65 -23 -80 2 -96 44 52 28 -28 4 10 -84 -10 -94 -47 23 -29 97 -92 40 41 100 18 -72 5 -74 97 7 4 32 -48 67 14 -72 74 57 -74
62 -33 -29 -69 -29 -24 38 73 -66 7 71 -70 -37 -32 79 86 -45 -92 43 -38 -11 -56 76 -31 6 10 -13 16 -32 -41 -55 -35 93 36 -37 -97 39 65 -11 36 71 -78 42 -36 -43 74 -83 79 -24 -75 -9 2 59 -11 -62 33 4 -4 24 0 -40 26 70
72 83 8 -1 50 -75 -54 -3 60 -14 24 70 -93 -36 -47 44 -89 -28 9 2 -14 -68 -12 -83 13 -20 -26 -81 -10 -73 -33 -93 96 46 -90 60 70 58 -57 -44 -44 95 17 -7 -5 94 74 -39 44 -14 47 52 46 -41 -67 -38 -48 -63 57 -93 -7 -88 -31 100 11 38 90 -50 -66 97 -59 41 -44
75 -47 5 -82 -100 -7 10 92 10 -29 -56 -81 -36 -75 18 -29 -23 -48 62 72 -54 59 -64 -22 -76 50 -56 87 35 -50 -97 -76 -16 -14 98 60 41 49 -79 7 -64 7 -23 -45 12 85 -18 -38 83 76 -16 -73 -34 -63 23 40 -66 5 -74 92 -82 36 -18 -28 69 39 -75 -53 -14 -80 60 61 74 71 -25 -78
47 68 -64 -19 78 -92 87 -59 54 -58 -91 -22 -97 -9 -81 -97 43 -83 27 73 -90 -47 -27 55 -27 92 26 61 -78 -27 -53 77 -23 -9 -38 9 -21 86 38 16 -60 48 -73 26 -3 69 65 30
59 64 -32 91 2 88 94 57 -88 -44 34 89 68 -47 58 -7 67 68 80 -15 23 7 6 -91 -97 0 26 -92 -43 -41 87 57 -49 65 -24 34 57 -36 -16 50 -17 -81 30 82 -37 -72 98 48 -10 46 -87 20 81 -47 96 99 -46 -54 -80 8
52 30 0 -76 -68 62 68 30 73 24 42 41 70 -15 30 81 -5 84 9 -6 53 7 7 -9 21 51 99 -71 92 -87 48 -38 82 -51 -41 77 35 97 -27 -79 -4 -98 -71 95 53 27 -46 30 -13 -79 -16 -5 -20
44 53 -51 -93 -10 -27 -18 -29 -24 -81 80 -65 74 86 100 53 22 73 11 86 20 78 -52 -33 -85 5 92 -41 54 87 42 86 78 -38 51 -28 19 18 41 100 -53 11 -43 -18 -14
94 -70 -12 0 19 18 -28 6 -75 -92 88 64 54 -10 -53 67 -56 55 -93 23 97 -53 -66 50 26 12 87 86 -5 42 25 6 45 48 -68 -54 -54 -50 -32 95 56 -86 34 67 8 68 100 -56 -79 97 -43 -60 43 95 -68 29 96 0 -55 -36 -73 44 81 67 -90 47 -64 26 -8 57 73 -62 -77 -22 56 27 60 -85 39 -76 67 -38 12 -29 -13 -99 36 40 -35 79 9 41 -45 -98 27
12 -81 83 -99 -68 -6 -5 74 -68 -70 -16 -28 -97
70 19 -9 40 -1 89 -47 -81 -65 40 65 27 -76 99 32 61 -33 -38 -38 -52 71 10 24 5 -41 -4 25 82 -28 -81 -41 -83 -61 -26 -64 96 -40 17 -48 -21 38 65 80 13 -64 -82 18 73 -42 75 -63 36 -95 -61 14 -57 68 -62 -37 -58 74 -5 0 -88 17 100 0 86 -38 -34 -50
47 73 82 -24 1 61 -49 -20 -50 -7 -77 -87 73 -2 85 51 -84 -4 35 -74 -79 44 -95 46 -96 -45 -13 15 -43 -48 100 -17 58 45 -2 33 69 31 -52 -74 31 75 -32 50 -47 -15 -87 63
29 75 90 100 -35 58 62 10 52 21 -86 -73 84 -96 -80 -95 12 99 -51 -8 -62 28 -83 -35 -34 87 38 54 -33 -73
21 57 89 91 -28 -23 34 -99 60 -28 -26 55 -25 64 -57 59 -83 -42 72 79 -5 -60
34 -59 7 -47 -85 -49 -79 63 -16 -45 -13 -65 36 27 -64 -25 -92 72 -19 -72 90 25 9 83 1 73 -25 -77 -33 4 65 46 54 44 51
51 -94 99 -92 -17 88 42 64 41 -37 -96 -91 -60 33 -19 -7 73 1 -37 70 34 80 -31 -94 12 35 -96 10 -75 55 -41 -48 66 -93 -44 -99 87 -7 -46 77 52 -41 90 -91 -7 -38 -55 21 53 -35 44 -52
26 36 -36 66 43 6 -34 6 -40 -44 25 25 -79 -65 43 82 -6 56 -64 67 -21 18 92 -80 -29 -57 1
44 89 47 90 -86 47 8 66 -89 92 -62 -80 29 64 -66 45 51 -77 26 -30 -68 -47 -5 10 -83 -27 -14 -40 -58 -29 66 -45 -79 93 -65 38 97 -41 84 95 9 52 -39 -79 -52
60 87 -38 46 -4 -4 25 100 -51 82 62 3 -35 25 -3 71 -76 -77 80 -91 41 -88 -14 -65 -22 67 -63 -53 -59 0 47 -60 66 -85 -11 -1 35 66 -12 71 -70 -85 64 -86 21 -1 31 22 12 -7 -75 -23 -14 46 52 3 -78 -12 19 -53 -21
35 -10 84 -60 -95 -19 -50 -20 -69 -51 -61 89 -70 -41 -46 -22 -33 43 42 -53 39 37 67 -17 -11 -64 -70 91 -44 -32 2 30 -90 -39 92 -80
79 -42 11 75 33 77 74 99 -33 22 -12 35 -56 -92 37 80 18 -43 -33 -65 66 90 0 65 -53 0 -84 39 9 22 9 -59 -67 47 -88 18 29 -8 27 -8 23 -4 39 -52 -1 -13 -88 19 32 31 -59 37 -54 26 -95 90 19 -95 -46 4 -75 -6 -69 29 -63 77 25 -81 -31 -69 -31 80 71 26 -22 87 72 13 11 16
79 79 -49 -100 61 97 -41 36 92 -84 -68 -16 41 -65 87 -41 -97 49 -91 -14 64 -15 -54 56 -4 57 -2 48 68 -74 6 19 41 93 -74 -72 -47 4 84 85 -79 5 -62 4 76 90 4 15 28 6 22 -94 -9 80 35 -86 48 6 -65 -19 -65 95 7 3 -37 -93 -82 -64 -4 -44 27 -94 -74 -98 93 48 -25 27 -84 76
34 31 7 5 -63 -68 -56 -55 -4 99 67 96 -20 59 84 42 -28 -48 -21 -63 7 64 91 -64 69 -52 -45 60 -89 -87 84 63 -55 68 -47
63 -90 94 -4 45 -92 -27 -83 61 -41 3 -83 61 -68 0 -33 -18 57 90 51 82 31 -5 -47 14 -35 -66 100 54 69 -67 90 -13 -56 52 -6 -26 -76 -99 26 -18 -56 5 -17 24 90 45 22 -97 99 -83 65 -98 -92 11 -24 -37 44 70 50 -94 -2 68 11
53 -73 -19 -100 -6 -41 -84 -97 98 58 -71 -44 11 100 -9 -54 -32 -53 40 66 -83 38 -15 16 -31 -20 -32 -32 -42 81 -1 -17 -49 6 17 35 28 -32 9 -74 -40 -68 -20 -5 -17 -44 -71 94 84 64 -37 50 100 44
33 37 -48 16 81 -34 74 -11 -60 84 -92 59 -17 -15 56 -69 44 100 -46 -76 41 -41 16 6 69 -11 72 73 37 0 6 -59 75 31
15 -78 -48 -75 -61 82 52 95 -88 72 -35 89 14 -85 -53 -42
75 53 -14 87 95 73 55 -46 -51 -37 -88 -25 43 -85 70 16 11 28 -8 80 38 90 -71 14 -31 62 -94 -95 88 59 -58 -2 65 -50 -60 79 34 -12 -91 -32 80 14 57 72 33 53 7 -46 -49 -56 18 -68 57 86 48 -28 -86 48 76 -40 -43 -45 54 98 -55 -66 79 51 14 31 -82 20 -99 17 45 37
97 69 0 2 -11 6 -60 25 10 -60 -3 37 -47 -40 78 81 86 19 -91 -15 27 93 97 16 30 -64 -28 61 -94 -89 -72 72 -83 61 -29 -46 -100 -57 34 -100 -91 -7 20 -28 23 -57 -32 16 40 -72 -40 -83 6 8 61 89 45 -7 59 -83 -97 74 -53 -51 -44 -80 -14 94 -86 -43 -66 -26 93 -1 -61 -15 35 -61 -7 85 -6 -18 71 94 53 -79 3 -28 -40 40 40 -67 -15 -66 82 -42 -5 -29
63 58 41 71 -81 -36 -65 55 -56 29 -83 -34 -50 84 0 -95 15 98 -90 79 -38 36 -27 -26 -6 60 23 47 69 14 -26 53 -2 -7 67 61 -65 -35 92 -100 -97 24 -93 84 97 -28 -16 99 -68 83 -87 38 -43 92 47 -19 76 -83 76 11 -44 -13 -76 79
18 -50 99 -40 -48 45 -35 -27 63 87 -62 -75 11 -10 81 -30 -22 -72 58
11 -75 -86 -62 90 3 -96 -90 -83 98 16 16
50 -79 50 55 19 -83 -37 49 -33 78 -4 65 -49 8 25 -68 -84 33 -7 49 -27 34 58 50 6 -11 96 -75 -64 32 -61 -17 -44 -74 96 -75 -19 46 16 -99 94 -2 100 -36 -34 17 -15 13 -100 -76 86
99 -51 -99 15 43 -56 -35 12 24 82 19 73 -39 -32 -50 -93 -39 -77 -37 81 68 -6 44 3 -100 -68 82 53 -100 38 -45 -100 0 58 -89 7 38 -40 -78 14 -96 61 -50 -93 8 54 67 -9 -52 -40 -72 64 50 -30 21 -63 50 -14 70 -85 11 -26 67 -53 -41 56 92 64 79 -1 -44 77 -84 37 50 -34 -87 33 -46 76 -2 99 -9 -90 32 -41 40 69 45 -45 51 -68 74 -33 24 -50 -40 83 -41 -53
65 59 91 -31 -58 44 7 21 -20 74 86 65 -44 82 -72 -14 89 91 76 48 15 -64 85 -42 96 75 -62 -15 61 -44 60 -47 67 -95 67 81 -73 81 60 86 79 97 -39 -32 -76 69 31 75 72 -82 -42 -13 93 18 45 -42 -5 -48 -90 -38 90 82 18 91 20 -77
25 53 -23 6 -89 72 86 87 -1 -91 14 92 -79 47 60 100 38 -18 74 -72 -13 75 -44 7 -16 21
41 68 -48 -56 -2 90 -22 16 58 99 -37 42 -30 51 23 34 -13 -6 -33 -88 -70 -1 -92 49 51 8 89 -98 -26 63 43 99 -64 95 -94 52 -63 39 -24 -36 21 21
71 -3 -61 90 62 -23 12 52 -31 3 81 -76 77 59 -48 34 -47 -38 36 -60 82 85 -56 -47 14 -97 91 51 84 11 32 -28 7 54 24 -57 -66 96 47 7 32 -82 -46 -19 -61 -98 4 -80 -25 13 97 -41 -99 -3 -84 65 -32 -4 85 50 -51 48 -95 -22 32 -39 21 91 -75 63 -78 43
2 73 -74
6 18 23 -14 88 -80 21
71 -20 -87 -63 61 19 75 46 56 -46 -40 93 41 -91 -1 -38 -78 59 82 -100 2 -18 -23 -2 73 72 -72 15 -61 96 11 -97 -87 -11 -72 26 8 -44 37 -44 27 2 -88 80 28 -100 4 -46 -10 -83 51 -79 26 50 -68 91 -76 68 -45 18 -76 -24 18 -1 -66 -34 6 -46 -5 -3 -72 -78
55 -95 -94 -84 21 8 -94 -39 -84 47 68 -33 42 37 57 93 54 -56 92 91 -26 -99 -6 75 12 87 -46 50 -99 2 86 18 -66 -92 47 -12 42 -11 -44 44 -61 -91 59 92 -91 -27 -72 99 -11 46 -37 30 -10 17 97 -76
27 -21 68 77 28 -29 -39 -56 52 0 -95 -72 21 -2 -85 83 72 -79 -94 -2 -18 -49 -21 -64 72 45 4 -49
49 91 33 97 17 72 -56 64 55 88 96 -86 -38 31 99 30 -30 78 35 5 99 4 24 67 11 0 34 -2 9 74 -64 35 14 -69 13 95 25 -52 4 15 -23 30 96 -33 80 -70 36 86 76 -81
30 27 35 67 -55 47 -65 -73 30 52 76 -48 64 -44 -21 38 36 81 -38 29 33 -63 4 75 39 -99 95 86 57 93 34
40 34 -94 -11 -59 -31 -47 -39 -67 15 13 46 -68 -82 87 -57 42 65 -47 -76 75 -44 29 -79 17 74 -7 94 97 84 -55 -87 30 -96 0 16 -71 36 64 -33 81
98 -46 40 53 -65 26 41 29 -89 17 -52 -25 14 61 -24 -62 -51 65 -32 20 -91 61 -88 -48 -98 86 93 -32 -53 63 47 28 -14 7 59 -84 -65 -30 51 37 5 -21 8 38 5 -52 -56 -31 -28 -54 -74 1 62 -4 -39 4 17 36 26 26 52 -10 -84 8 41 27 -62 64 96 70 -56 53 -70 -99 -23 79 78 100 99 80 62 -5 -24 -91 84 69 79 -92 46 50 -21 -26 -50 18 -38 41 64 73 -57
66 -75 66 49 -76 76 24 -86 -97 0 84 -100 -88 -63 97 28 -13 -5 -8 -22 0 5 25 8 -79 -13 -73 -93 41 -49 -79 -27 93 -65 66 -100 -39 4 68 -31 -71 -63 -65 -39 53 0 -5 -55 28 89 99 43 -43 42 -17 -29 18 16 93 -59 23 -39 -36 94 91 -62 4
19 -53 14 41 21 -4 -54 -78 -79 35 62 20 43 -2 -10 -24 19 -58 -31 -19
63 -88 17 56 -55 15 -16 -88 99 -8 -37 10 8 29 57 20 18 5 45 4 -32 -4 -28 -32 -78 -60 67 16 76 -45 52 -27 -71 3 96 -91 -47 -7 31 -99 -4 -91 38 64 -26 -8 -52 -67 72 -23 41 26 -66 58 28 -13 -27 -17 45 -57 39 -91 -77 -8
44 0 -63 -32 -37 27 67 -59 -53 -100 100 -57 80 -87 12 -73 2 -90 76 -35 -8 66 -51 29 -58 19 -63 44 28 -64 17 58 -17 -76 -25 79 -92 -61 -34 99 96 -10 -55 -77 87
94 18 -39 21 -80 38 45 -40 -80 -53 8 -62 69 71 -36 -14 74 11 -82 -92 95 27 -84 -43 74 -10 45 43 64 -66 -38 25 47 -19 21 22 56 -95 19 3 85 -67 -86 -31 49 93 47 -36 50 88 94 31 85 -53 32 -17 31 29 -61 -88 37 78 62 -34 55 -83 30 -22 100 92 -5 72 51 -12 57 4 -85 15 -92 -46 -28 -92 -58 -71 -72 -43 43 3 65 65 18 54 -9 -74 -62
69 53 91 53 -44 -20 -54 -1 69 -50 -37 5 26 -6 81 5 -85 -58 30 58 -44 61 -83 90 -67 -4 1 53 14 53 95 32 42 -56 -92 68 38 -90 19 79 12 84 66 8 -64 -21 78 69 -25 1 -41 90 -46 -31 28 67 -78 77 -24 -39 80 80 41 82 81 67 -95 -83 -32 64
51 78 55 21 -53 80 18 20 86 95 -31 -16 -1 -100 -56 -41 36 59 33 -63 94 16 15 10 83 -99 92 -9 79 18 -11 -63 58 -10 15 -14 -86 55 -68 -84 -100 66 11 -1 10 30 -75 4 67 69 99 -29
15 33 78 -82 16 15 -30 40 3 94 -76 -77 12 -34 -20 35
71 12 -74 -6 23 -35 -83 -6 39 31 84 20 59 -62 85 49 -17 23 15 -4 59 -51 42 -20 29 69 -17 -100 11 -77 38 100 -50 -99 3 -71 -84 71 -66 -17 -23 -83 -69 -99 22 -49 -31 40 45 -6 -70 77 40 19 -88 73 41 -53 -65 -47 21 54 71 22 39 -24 67 -54 19 -85 5 -32
72 4 47 -1 80 -99 -83 -44 28 -38 -29 49 99 92 86 65 16 -33 -42 54 36 6 -74 -53 -88 69 27 -7 -37 18 -9 -34 44 14 53 70 -82 58 26 80 -23 -59 -52 93 -63 44 61 -32 17 -46 91 46 60 -91 26 -65 -14 -42 89 7 -71 -55 9 76 -25 32 -41 -4 -77 -22 -76 57 36
67 19 -65 0 68 91 23 -40 -13 77 60 61 92 -82 48 68 -88 0 35 -40 76 99 -38 88 80 82 -47 -39 -17 21 -21 -1 1 62 83 6 -46 -19 8 -4 0 26 96 -46 12 -87 55 -40 42 -54 -40 -55 -17 98 -42 96 46 43 36 16 -61 -68 4 70 -56 14 -22 -61
58 72 51 -8 51 17 -20 -12 -100 -98 27 75 79 -24 85 15 81 63 -86 -73 4 22 -53 21 82 31 51 39 70 49 -49 79 -63 60 85 -65 -61 95 34 -99 21 52 -67 -88 73 -6 -68 -38 56 39 24 80 -73 -17 2 37 23 -39 -47
41 -45 35 59 -11 78 33 -100 55 69 -68 7 -65 -49 -97 -24 -60 -82 28 -22 87 -18 25 96 19 -23 100 -97 37 9 -97 -35 10 6 25 -8 -2 -13 23 -36 34 93
58 95 44 27 -38 -17 17 94 90 -10 -94 56 55 49 -67 -19 51 -19 16 -75 -93 25 96 -67 -89 -14 -92 77 7 -95 66 51 -3 -75 -14 33 -11 -83 80 29 -55 89 81 -62 -67 -33 48 -78 -28 -59 -34 -73 100 -100 99 -20 89 -99 36
27 -59 -32 38 -56 41 -3 -31 56 3 26 3 -91 -34 10 -96 55 -89 23 88 85 91 -83 76 -87 -35 74 42
37 -11 -58 -44 -40 10 4 44 80 69 -71 1 -76 -4 -36 18 -20 -69 40 -43 10 93 70 75 71 -72 50 -84 69 -29 75 -24 48 49 19 -95 0 -15
41 96 35 -27 34 -31 88 42 83 -45 43 44 82 28 -77 -85 -74 76 54 -57 -27 -56 -93 53 34 -37 51 60 70 89 -97 -39 62 -84 -50 84 -41 70 35 -14 -43 -96
58 3 -45 -65 -92 3 31 -23 16 33 -14 -46 2 -59 18 56 60 11 -37 16 18 1 62 -80 -78 -78 61 -95 13 -42 83 94 95 -12 -51 -13 11 49 36 -30 -1 -73 66 17 94 29 -46 -31 -8 70 -43 -69 -93 -32 14 94 -86 -63 -52
10 27 70 -61 6 -38 82 51 78 -49 42
56 -78 -100 -89 -59 88 71 41 53 -97 -60 -51 -85 -7 8 28 -54 -50 53 -90 -55 67 43 -31 -15 86 48 68 -66 93 19 -77 62 -80 -42 -63 17 31 -17 97 -97 33 -53 -34 -18 -55 13 9 -32 -53 99 -30 -60 -78 19 -96 57
69 12 26 -81 -67 -72 -40 -69 43 16 91 -90 28 -99 35 -76 -4 27 -18 -9 41 7 37 12 -9 71 16 -71 -16 -75 16 66 89 15 56 0 4 -92 -27 -63 1 10 83 27 33 -72 20 -46 2 30 -15 -46 12 18 98 73 53 24 -65 91 12 11 94 18 -72 -53 85 -46 18 -57
16 -76 49 99 -75 16 -43 -71 -8 30 8 52 -15 37 -50 -47 58
39 59 -1 11 62 54 -95 -57 -42 85 99 -24 99 10 -92 93 100 -35 31 54 -77 -97 -50 -48 -60 -98 -20 -8 -98 86 70 22 -99 -30 -3 51 100 22 -91 64
73 16 70 -100 82 18 -37 -74 77 -14 -85 -13 -19 49 -26 38 31 92 -35 -5 -86 33 52 -34 -56 -47 -28 46 -69 -88 -30 67 29 33 39 -26 -79 29 72 82 -79 35 4 36 44 -74 53 -48 -78 65 72 -47 72 11 -1 -59 -62 -36 -31 20 61 95 10 72 25 58 -94 -24 54 7 -30 43 89 86
40 -4 -58 88 -48 -59 -17 30 58 59 72 19 -58 -8 3 11 8 -37 -9 52 -91 37 57 -94 49 35 28 49 81 22 29 -77 0 18 -36 -16 30 -16 96 -40 53
52 5 -91 -98 29 -68 72 55 51 -70 76 -97 -83 -73 -55 84 -26 -62 45 -39 -15 95 -83 47 100 -85 -10 0 -1 -30 -17 80 -37 -22 32 -70 16 25 -84 4 -16 91 62 -3 -80 79 -43 30 -30 46 -20 60 -35
65 9 -99 1 21 55 -92 53 -18 73 77 4 -37 58 -44 61 -42 -70 30 23 -49 74 29 -8 -88 84 94 86 -69 6 1 9 4 96 -4 -40 41 -89 50 11 86 89 14 37 54 18 -63 -29 29 59 -19 -56 -82 -71 -63 -6 1 -83 -90 62 -68 19 27 -35 1 -63
90 -72 68 -86 -9 86 39 -82 -14 88 -64 -98 0 30 30 -80 -3 -32 49 -18 -51 -89 -48 55 -25 -5 -84 83 44 -31 -28 -62 -65 -63 -5 -57 -76 -43 41 -67 -97 -69 -21 -41 44 -6 -51 8 97 -25 30 53 -9 -13 2 55 -46 -7 -31 43 69 89 58 8 -28 -14 -1 12 88 -25 -59 -100 71 44 44 54 -21 16 40 -99 -64 7 -54 -24 -39 -29 -11 -3 22 -18 -56
41 58 49 -36 -50 -20 -81 -29 72 66 -27 42 -1 -95 75 -41 6 -60 96 -20 73 -27 87 -65 -54 -61 29 20 31 -6 -57 29 -77 -39 -17 54 -39 86 14 -57 -100 -94
78 8 93 -35 -4 62 4 13 -78 84 -14 -56 6 -42 84 24 30 -65 67 -80 88 97 75 10 56 40 53 -95 32 100 14 100 -27 -52 -83 -1 49 20 -48 67 -25 -41 77 -36 34 4 42 -74 -79 -70 -63 61 -13 -82 -74 100 -93 11 -25 60 -40 -24 -34 91 98 -84 -12 60 15 2 -80 -93 -22 89 -15 -51 66 -45 -51
17 94 71 -59 52 -70 -65 64 -49 100 -73 -61 55 -40 66 -84 90 93
15 -63 -85 45 -77 67 -12 0 -47 25 17 88 68 -77 -27 47
38 -7 -36 83 -64 -7 69 17 21 -71 25 -18 -98 71 -45 -89 19 -48 78 24 -99 -11 59 -84 50 -58 -14 71 90 35 14 -99 -82 67 -60 -44 -73 -30 26
45 18 23 17 36 9 -47 100 41 -55 52 18 -38 10 55 -3 18 -82 71 -57 51 -47 24 100 100 -29 4 -78 92 -88 47 8 25 100 37 55 83 16 63 -92 97 -73 -100 51 90 -94
41 87 89 -65 -19 100 84 30 -21 -9 31 -67 -100 43 70 -26 35 74 46 65 -15 -79 7 -2 19 13 64 4 -9 11 -42 27 82 -17 83 24 -54 41 84 52 52 -24
78 15 100 -45 18 -28 91 -48 11 -35 49 60 90 100 -71 -44 -82 10 26 58 -67 -51 -21 27 -82 83 98 -56 31 75 33 91 51 -50 73 1 -89 18 -54 31 -37 -12 97 61 0 -77 76 45 30 36 18 -44 -50 -44 49 28 -32 95 5 74 80 -14 -93 -92 12 84 -45 81 -54 -84 5 -30 -58 -63 0 29 8 90 24
80 -55 -10 78 -10 -14 -54 96 100 -90 -95 81 -56 -87 14 18 -53 -39 -81 86 86 -72 -79 -82 -9 43 -51 -90 68 -33 -33 -92 -62 -34 -7 58 2 30 -78 40 -45 2 -17 -9 91 89 6 -13 -99 -76 4 58 -43 -38 23 34 -30 -7 84 -80 69 85 -91 -60 66 53 94 73 -43 88 -48 80 3 -77 27 63 19 71 -11 91 93
57 -13 20 -45 1 95 67 -98 -52 -84 -59 41 86 71 -29 95 74 -15 -74 67 -33 -32 -72 30 50 32 -75 -12 -31 -91 79 -97 28 -92 -63 -86 -9 44 -15 -70 100 10 -30 44 99 56 -50 -28 73 -42 -13 97 67 14 -95 -18 19 63
27 -17 -52 19 72 -29 -27 16 -67 -87 -44 -21 -80 -21 24 22 -90 38 -36 -37 14 -22 -16 41 89 43 -98 67
23 -13 63 -14 -5 -28 -40 80 -15 58 26 68 29 98 95 92 -43 8 -8 27 -30 -65 84 -29
