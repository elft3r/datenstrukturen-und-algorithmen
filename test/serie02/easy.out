728
394
1276
301
410
338
237
198
569
282
448
664
1092
295
200
499
266
286
799
960
1014
746
83
259
317
433
249
410
235
267
362
366
175
448
409
434
418
399
409
281
584
421
485
150
130
281
444
1359
481
330
549
73
115
305
546
173
1295
609
359
811
413
160
250
195
948
967
519
167
450
640
919
673
292
368
287
420
472
394
215
312
543
148
420
595
414
192
704
502
246
671
184
206
210
754
792
815
669
397
173
531
